docs/integrations/databases/opentelemetry/cassandra-opentelemetry.md

```diff
@@ -0,0 +1,169 @@
+---
+id: cassandra-opentelemetry
+title: Cassandra - OpenTelemetry Collector
+sidebar_label: Cassandra - OTel Collector
+description: Learn about the Sumo Logic OpenTelemetry App for Cassandra.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/databases/cassandra.png')} alt="Thumbnail icon" width="75"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
+
+The [Cassandra](https://cassandra.apache.org/_/cassandra-basics.html) - OpenTelemetry app is a log based app that helps you monitor the availability, performance, health, and resource utilization of your Cassandra clusters. Preconfigured dashboards provide insight into resource utilization, cache/Gossip/Memtable statistics and Error and warnings. Cassandra logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Cassandra-OpenTelemetry/Schematics.png')} alt="Schematics" />
+
+## Fields creation in Sumo Logic for Cassandra
+
+Following are the [Fields](https://help.sumologic.com/docs/manage/fields/) which will be created as part of Cassandra App install if not already present:
+
+- **`db.cluster.name`** - User configured. Enter a name to identify this Cassandra cluster. This cluster name will be shown in the Sumo Logic dashboards.
+- **`db.system`** - Has fixed value of **cassandra**.
+- **`deployment.environment`** - User configured. Through this Cassandra cluster is identified by the environment where it resides. For example: dev, prod or qa.
+- **`sumo.datasource`** - has fixed value of **cassandra**.
+
+## Prerequisite
+
+The app supports Logs from the open-source version of Cassandra. The App is tested on the 3.11.10 version of Cassandra.
+
+Cassandra has three main logs: system.log, debug.log, and gc.log which hold general logging messages, debugging logging messages, and java garbage collection logs respectively.
+
+These logs by default live in ${CASSANDRA_HOME}/logs, but most Linux distributions relocate logs to /var/log/cassandra. Operators can tune this location as well as what levels are logged using the provided logback.xml file. For more details on Cassandra logs, see[ this](https://cassandra.apache.org/doc/latest/troubleshooting/reading_logs.html) link.
+
+## Configure Cassandra Logs Collection
+
+### Step1: Set up Collector
+
+:::note
+If you want to use an existing OpenTelemetry Collector, you can skip this step by selecting the **Use an existing Collector** option.
+:::
+
+If you want to create a new Collector please select **Add a new Collector** option.
```

Comment:
```suggestion
To create a new Collector:

1. Select the **Add a new Collector** option.
2. Select the platform for which you want to install the Sumo OpenTelemetry Collector.

This will generate a command which can be executed in the machine which needs to get monitored. Once executed it will install the Sumo Logic OpenTelemetry Collector agent.
```